PayPalDataCollector/src/main/java/com/braintreepayments/api/PayPalDataCollectorInternalRequest.java

```diff
@@ -7,17 +7,25 @@
 /**
  * Used to configuration the PayPalDataCollector request
  */
-class PayPalDataCollectorRequest {
+class PayPalDataCollectorInternalRequest {
 
     private String applicationGuid;
     private String clientMetadataId;
     private boolean disableBeacon;
+    final private boolean hasUserLocationConsent;
     private HashMap<String,String> additionalData;
 
+    /**
+     * @param hasUserLocationConsent Indicates if the beacon feature should be disabled.
+     */
```

Comment:
```suggestion
```

can skip for internal class - or update for hasUserLocationConsent

Reply:
👍 it's also the incorrect comment 😅